src/server.py

```diff
@@ -24,6 +24,15 @@ def api_top_image():
         "title": article.title,
         "topimage": article.top_image}), 200, {'Content-Type': 'application/json'}
 
+
+@app.route('/text',methods = ['POST'])
```

Comment:
I think the endpoint should be called something like /fulltext to support potential difference between /text and /fulltext

Reply:
renamed